README.rst

```diff
@@ -1,76 +1,61 @@
-=======
-Astropy
-=======
-
-.. container::
+|Astropy Logo|
 
-    |Actions Status| |CircleCI Status| |Coverage Status| |PyPI Status| |Documentation Status| |Pre-Commit| |Ruff| |Zenodo|
+|User Stats|
 
-The Astropy Project (https://astropy.org/) is a community effort to develop a
-single core package for Astronomy in Python and foster interoperability between
-Python astronomy packages. This repository contains the core package which is
-intended to contain much of the core functionality and some common tools needed
-for performing astronomy and astrophysics with Python.
+----
 
-Table of Contents
-=================
+|Actions Status| |CircleCI Status| |Coverage Status| |PyPI Status| |Documentation Status| |Pre-Commit| |Ruff| |Zenodo|
+----
+The Astropy Project is a community effort to develop a
+single core package for astronomy in Python and foster interoperability between packages used in the field.
+This repository contains the core library.
 
-- `Installation <#installation>`_
-- `Contributing <#contributing>`_
-- `Getting Started with GitHub Codespaces <#getting-started-with-github-codespaces>`_
-- `Supporting the Project <#supporting-the-project>`_
-- `License <#license>`_
+* `Website <https://astropy.org/>`_
+* `Documentation <https://docs.astropy.org/>`_
+* `Slack <https://astropy.slack.com/>`_
+* `Open Astronomy Discourse <https://community.openastronomy.org/c/astropy/8>`_
+* `Astropy users mailing list <https://mail.python.org/mailman/listinfo/astropy>`_
+* `Astropy developers mailing list <https://groups.google.com/g/astropy-dev>`_
+* `Astropy community engagement study <https://zenodo.org/records/10603049>`_
+* `Astropy diversity, equity and inclusion study <https://astropy-dei.orgmycology.com/>`_
 
 Installation
 ============
 
-Releases are `registered on PyPI <https://pypi.org/project/astropy>`_,
-and development is occurring at the
-`project's GitHub page <https://github.com/astropy/astropy>`_.
-
-For detailed installation instructions, see the
-`online documentation <https://docs.astropy.org/>`_
-or `docs/install.rst <docs/install.rst>`_ in this source distribution.
-
 To install `astropy` from PyPI, use:
 
 .. code-block:: bash
 
     pip install astropy
 
+For more detailed instructions, see the `install guide <https://docs.astropy.org/en/stable/install.html>`_ in the docs.
```

Comment:
Some inconsistent word wrapping throughout -- maybe wrap it to 80 chars (I think?) like some other lines here?

Reply:
Fixed in 
868fa26